src/main/java/com/contrastsecurity/maven/plugin/ContrastException.java

```diff
@@ -0,0 +1,30 @@
+package com.contrastsecurity.maven.plugin;
+
+/** Indicates an error occurred while using the Contrast HTTP API */
+public class ContrastException extends RuntimeException {
```

Comment:
The Contrast SDK currently conflates all 400 series errors as an auth issue. I introduced this for errors that don't fall into the "auth issue" category